nimble/host/src/ble_eatt.c

```diff
@@ -75,7 +76,14 @@ static struct ble_gap_event_listener ble_eatt_listener;
 static struct ble_npl_event g_read_sup_cl_feat_ev;
 
 static void ble_eatt_setup_cb(struct ble_npl_event *ev);
-static void ble_eatt_start(uint16_t conn_handle);
+
+struct os_callout eatt_conn_timer;
```

Comment:
this needs to be per con, and should be compiled in only when peripheral is supported

Reply:
Done